planning/static_centerline_optimizer/test/test_static_centerline_optimizer.test.py

```diff
@@ -43,6 +43,10 @@ def generate_test_description():
             {"lanelet2_output_file_path": "/tmp/lanelet2_map.osm"},
             {"start_lanelet_id": 215},
             {"end_lanelet_id": 216},
+            os.path.join(
+                get_package_share_directory("mission_planner"),
+                "config/mission_planner.param.yaml",
```

Comment:
```suggestion
                "config",
                mission_planner.param.yaml",
```
better for multi-platform